### ISSUE TITLE
generating samples by the pretrained model 

### ISSUE THREAD

**Issue by Mathilda88** [Zhangjinso/PISE#34]
Hello,

Thanks for sharing the great work. I'm trying to generate the samples using the pretrained model. But unfortunately my results are so dim, something like this:
Mine:
![fashionMENJackets_Vestsid0000488201_7additional_fashionMENJackets_Vestsid0000488201_1front_vis](https://user-images.githubusercontent.com/74356717/133560468-e8b3e306-74d6-4fed-b34f-ae1381767ee8.jpg)
Previously available by the authors:
![fashionMENJackets_Vestsid0000488201_7additional_fashionMENJackets_Vestsid0000488201_1front_vis](https://user-images.githubusercontent.com/74356717/133560929-2ab96226-24a2-4469-b642-459e6488c852.jpg)

Any help is greatly appreciated by the entire community.

**Comment by Zhangjinso** [comment]
Hi, it is strange to get this kind of outputs. Could you give more information about your procedures for this output? Like the platform you used and the outputs after running...

**Comment by Mathilda88** [comment]
Thanks for your response. Actually, I'm on Fedora Linux, I tried different types of GPUs on a cluster, RTX8000 and V100 but the results are quite the same, with an old_size (256,256) and all the default setting of parameters.  Additionally, I'm using the following packages:

python=3.6, pytorch=1.2, cudatoolkit=10.0 torchvision,
anaconda                0.0.1.1
bleach                  1.5.0
block-extractor-cuda    0.0.0
certifi                 2021.5.30
cffi                    1.14.6
cycler                  0.10.0
dataclasses             0.8
decorator               4.4.2
dominate                2.6.0
html5lib                0.9999999
imageio                 2.9.0
importlib-metadata      3.4.0
kiwisolver              1.3.1
local-attn-reshape-cuda 0.0.0
matplotlib              3.3.4
mkl-fft                 1.3.0
mkl-random              1.1.1
mkl-service             2.3.0
natsort                 7.1.1
networkx                2.5.1
numpy                   1.19.2
olefile                 0.46
opencv-python           4.5.3.56
pandas                  1.1.5
Pillow                  8.3.1
pip                     21.0.1
pycparser               2.20
pyparsing               2.4.7
python-dateutil         2.8.2
pytz                    2021.1
PyWavelets              1.1.1
resample2d-cuda         0.0.0
scikit-image            0.17.2
scipy                   1.5.4
setuptools              58.0.4
six                     1.16.0
tensorflow-tensorboard  0.4.0
tifffile                2020.9.3
torch                   1.2.0
torchvision             0.4.0a0
tqdm                    4.62.2
typing-extensions       3.10.0.0
wheel                   0.36.2
zipp                    3.4.1

**Comment by Mathilda88** [comment]
Here are some examples:
![fashionWOMENBlouses_Shirtsid0000546609_3back_fashionWOMENBlouses_Shirtsid0000546609_2side_vis](https://user-images.githubusercontent.com/74356717/133824209-be22058d-acf2-42a0-9586-7fc88c8ac092.jpg)
![fashionWOMENBlouses_Shirtsid0000750504_7additional_fashionWOMENBlouses_Shirtsid0000750504_2side_vis](https://user-images.githubusercontent.com/74356717/133824221-43af2d5e-782d-4475-bd0f-5a7ff9449fc9.jpg)
![fashionWOMENBlouses_Shirtsid0000787301_1front_fashionWOMENBlouses_Shirtsid0000787301_4full_vis](https://user-images.githubusercontent.com/74356717/133824224-39714ebb-8ddb-4ec3-b0aa-63be8c127b19.jpg)
![fashionWOMENCardigansid0000011903_1front_fashionWOMENCardigansid0000011903_2side_vis](https://user-images.githubusercontent.com/74356717/133824232-25ee470b-c7a3-443e-a6dc-d8e7e1d3e2d6.jpg)

**Comment by Zhangjinso** [comment]
Hi, I just test the current code with the checkpoints (with titan x on ubuntu), and the results are ok.
I have never been seen this kind of result before. Sorry

**Comment by Mathilda88** [comment]
Thanks for your response. May you keep the topic open to get the chance of leveraging from other experiences.

**Comment by Jackson-1102** [comment]
Hello, I would like to ask that whether the FID Scores is tested by the test_result you provided ?

**Comment by Zhangjinso** [comment]
Yes. The fid is computed as 256X256 between the generated images and the training images using [this](https://github.com/mseitzer/pytorch-fid/tree/master/src/pytorch_fid).
